### PR TITLE
Updated Lit-Review (new document)

### PR DESCRIPTION
**Pull request by Kirs10-Riley**
I created a new document because the old started to get unorganized.  I have added more papers and removed a few. I tried to keep track of all the papers I removed and created a section below of the ones I took out and a small reason explaining why. I added all the notes that was left in the last pull-request as well.

### CODE REVIEW

File: LIT-REVIEW.Rmd [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+
+-   Abstract: "Four political models (the international system, rational domestic economic policy, intra-govemmental politics, and interest group influence) may explain the cross-national structure of average nominal levels of tariffs on manufactures. Using available data measuring or approximating the explanatory variables for the time periods 1902 and 1971, selected hypotheses were tested in regression equations. The 1971 tariff levels are best predicted by rational economic policy variables while for 1902, international power variables provide the best predictions of the tariff. In general, the causal forces influencing the cross-national pattern of tariff levels appear to have shifted in the 20th century from those indicating international power to those measuring domestic politico-economic development. These results provide evidence relevant to some of the general propositions about size and development suggested by the contemporary political economy literature. They also lend support to those who argue that a rising level of international interdependence has resulted in long-term changes in the pattern of influences on national public policies."
```
Comment (kbuzard) [inline LIT-REVIEW.Rmd:50]: Can you give me a better idea of how detailed the data is? If it's all at the country level (no industrial/sectoral detail), please say that.

Also, we don't need the doi at the top repeated at the bottom.

File: LIT-REVIEW.Rmd [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+
+-   Notes: The author uses empirical data on tariff reforms. Kowalcyzk and Davis investigates NAFTA phase-outs in the United states and Mexico. They look at all import and and 9 subcategories of imports. Imperial work starts on page 234 and ends on page 245.
+
```
Comment (kbuzard) [inline LIT-REVIEW.Rmd:65]: Is this all about NAFTA? I can't tell what years the data covers. If the data doesn't start before 1988, we don't need this one.

File: LIT-REVIEW.Rmd [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+-   <https://doi.org/10.7208/9780226260228-012> or <http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.477.1901&rep=rep1&type=pdf>
+
+*LOOK INTO THE PAPERS IN 10*
```
Comment (kbuzard) [inline LIT-REVIEW.Rmd:68]: What is the status of this note?

File: LIT-REVIEW.Rmd [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+
+### 11. Golub, Stephen S., and J. M. Finger. "The Processing of Primary Commodities: Effects of Developed-Country Tariff Escalation and Developing-Country Export Taxes." Journal of Political Economy, vol. 87, no. 3, 1979, pp. 559--577. JSTOR, www.jstor.org/stable/1832023.
+
+\- Abstract: "Both DC (developed country) tariffs and LDC (developing country) export taxes are "escalated" to protect local processors of primary commodities. The paper develops an analytical model of north-south commodity trade which is used to estimate the effects of reciprocal elimination of these trade barriers for eight commodities. It is estimated that processing would increase by 9 percent in the LDCs and decline by less than 1 percent in the DCs. The LDC export revenue for the eight-commodity sample would increase by 11 percent, or just over \\\$1 billion (based on 1973 trade flows), which is considerably more than the estimated effect of the Generalized System of Preferences."
```
Comment (kbuzard) [inline LIT-REVIEW.Rmd:76]: How does this involve U.S. data? What years does it cover?

File: LIT-REVIEW.Rmd [hunk 5]
```diff
@@ -0,0 +1,250 @@
...
+
+-   Notes: US trade data on page 484, 489, and 497. He uses import data, export data, import/ export, net exports, production, and consumption for different factor content of US trade.
```
Comment (kbuzard) [inline LIT-REVIEW.Rmd:90]: What years? If there's no sectoral/industrial detail, please say that explicitly.

File: LIT-REVIEW.Rmd [hunk 6]
```diff
@@ -0,0 +1,250 @@
...
+
+-   Comment: This is a book, I would skip to the chapter called "some imperial evidence" can be found in the SU Library ISBN: 0262111594 9780262111591 or Bird 4th floor.
```
Comment (kbuzard) [inline LIT-REVIEW.Rmd:112]: here you have "imperial" but I think you mean "empirical." I've seen this a couple other places. If I'm right that this should be "empirical," please fix it here and anywhere else it shows up in the document

**Review by kbuzard — CHANGES_REQUESTED** [review]
@Kirs10-Riley Thanks for this update! I've gotten down to #22 and I'll have to finish the rest later. But I wanted to get these comments to you now.

File: LIT-REVIEW.Rmd [hunk 7]
```diff
@@ -0,0 +1,250 @@
...
+
+-   Intro: "This paper analyzes the economic efficiency effects of tariff concessions made in the Kennedy Round trade negotiations. The static welfare methodology used differs from previous analyses of multilateral tariff reductions by focusing on terms of trade changes. These changes often are incompletely allowed for because the underlying currency appreciations and depreciation necessary to balance unequal shifts in import and export values are ignored. Relative to the traditionally estimated welfare gains from tariff reductions, these terms of trade effects may have a major impact. It is shown that the Kennedy Round cuts may have left the U.S. worse off due to a terms of trade deterioration... Welfare effects from the Kennedy Round negotiations are calculated for four major industrial countries: France, Germany, Japan, and the U.S. The empirical results illustrate that seemingly comparable tariff reductions result in different efficiency effects across countries and that countries within a trading bloc are not affected identically" (pg. 760-761)
```
Comment (kbuzard) [inline LIT-REVIEW.Rmd:182]: Please add the years and whether or not there is sectoral/industrial detail.

File: LIT-REVIEW.Rmd [hunk 8]
```diff
@@ -0,0 +1,250 @@
...
+
+-   Intro: In this paper a numerical general equilibrium model of international trade involving major world trading blocs (the United States, Japan, the EEC and the Rest of the World) is used to analyse the effects of alternative tariff-cutting formulate proposed by the major participants in the Tokyo Round negotiations under the GATT at an earlier stage preceding the recent agreement which was initialized in April 1979. We also compare our findings with model experiments involving more extensive forms of world trade liberalisation. Multilateral tariff reductions corresponding to individual country proposals are considered along with a sequence of more wide-ranging trade liberalisation measures on multilateral, bilateral and unilateral bases. The model used is described elsewhere (Whalley, 1978a) and only a brief verbal summary is given here so as to keep the main focus of the paper on results."
+
+-   Notes: The product and industrial classification unites in the general equilibrium trade models are listed in table 1, page 843. The data used is from GATT (1974), Yeats (1976;1977)
```
Comment (kbuzard) [inline LIT-REVIEW.Rmd:188]: Is the data for the year 1974, or is this a reference to a GATT publication? Bottom line: I want to know what years of data they have.

Can you also add a few words about how detailed the product/industrial classification is? We want to be able to know at a glance what's going on without having to go get access to the paper.

File: LIT-REVIEW.Rmd [hunk 9]
```diff
@@ -0,0 +1,250 @@
...
+
+-   This is where I left off I have not looked at this paper yet.
+
+## Removed:
+
+1.  Edward E. Leamer, James Levinsohn, Chapter 26 "International trade theory: The evidence, Handbook of International Economics", Elsevier, Volume 3,1995, Pages 1339-1394, ISSN 1573-4404, ISBN 9780444815477burgees
+
+2.  Bergstrand, Jeffrey H. "The Gravity Equation in International Trade: Some Microeconomic Foundations and Empirical Evidence." The Review of Economics and Statistics, vol. 67, no. 3, 1985, pp. 474--481. JSTOR, www.jstor.org/stable/1925976.
+
+    1.  Reason: Paper focuses on a generalized gravity equation and the "emperical evidence" he used does not specify an exact country.
```
Comment (kbuzard) [inline LIT-REVIEW.Rmd:234]: please turn these indented "1." 's into bullets or similar--the numbering is confusing.

This explanations for why this was removed for the Bergstrand paper and 4 and 5 (below) are perfect.

File: LIT-REVIEW.Rmd [hunk 10]
```diff
@@ -0,0 +1,250 @@
...
+
+### 32. Irwin, Douglas A. (1994). The GATTs Contribution to Economic Recovery in Post-War Western Europe. <doi:10.3386/w4944>
+
+-   <https://www.federalreserve.gov/pubs/ifdp/1993/442/ifdp442.pdf>
+
+-   This is where I left off I have not looked at this paper yet.
+
+## Removed:
+
+1.  Edward E. Leamer, James Levinsohn, Chapter 26 "International trade theory: The evidence, Handbook of International Economics", Elsevier, Volume 3,1995, Pages 1339-1394, ISSN 1573-4404, ISBN 9780444815477burgees
```
Comment (kbuzard) [inline LIT-REVIEW.Rmd:230]: Explanation for why removed is missing.

File: LIT-REVIEW.Rmd [hunk 11]
```diff
@@ -0,0 +1,250 @@
...
+
+2.  Bergstrand, Jeffrey H. "The Gravity Equation in International Trade: Some Microeconomic Foundations and Empirical Evidence." The Review of Economics and Statistics, vol. 67, no. 3, 1985, pp. 474--481. JSTOR, www.jstor.org/stable/1925976.
+
+    1.  Reason: Paper focuses on a generalized gravity equation and the "emperical evidence" he used does not specify an exact country.
+
+3.  Chenery, Hollis B., and Lance Taylor. "Development Patterns: Among Countries and Over Time." The Review of Economics and Statistics, vol. 50, no. 4, 1968, pp. 391--416. JSTOR, www.jstor.org/stable/1926806.
+
+    1.  reason: trade is not the main focus of the paper.
```
Comment (kbuzard) [inline LIT-REVIEW.Rmd:238]: The explanation doesn't make clear whether there is NO trade data. If there is trade data in there, we might want it in the lit review.

File: LIT-REVIEW.Rmd [hunk 12]
```diff
@@ -0,0 +1,250 @@
...
+
+    1.  reason: uses data from 1880-1920
+
+5.  Baldwin, Robert E. "The Political Economy of Trade Policy." The Journal of Economic Perspectives, vol. 3, no. 4, 1989, pp. 119--135. JSTOR, www.jstor.org/stable/1942913.
+
+    1.  No empirical data just policy
+
+6.  Burgess, David F. "Tariffs and Income Distribution: Some Empirical Evidence for the United States." Journal of Political Economy, vol. 84, no. 1, 1976, pp. 17--45. JSTOR, www.jstor.org/stable/1830169.
+
+    1.  does not fit very well.
```
Comment (kbuzard) [inline LIT-REVIEW.Rmd:250]: This isn't a convincing explanation for excluding this paper. We need to know whether it uses data, and if so, whether that data is of interest to us (years of data, etc).

**Review by kbuzard — CHANGES_REQUESTED** [review]
Okay--I've finished this up. There are a few more questions, but overall this is in good shape. I think it's totally do-able to clean this up by the end of the week. What do you think @Kirs10-Riley ?

**Comment by Kirs10-Riley** [comment]
> Okay--I've finished this up. There are a few more questions, but overall this is in good shape. I think it's totally do-able to clean this up by the end of the week. What do you think @Kirs10-Riley ?

I'll have it done by this week.

File: LIT-REVIEW.Rmd (hunk 3 above)
Comment (Kirs10-Riley) [reply LIT-REVIEW.Rmd:68]: I forgot to deleted this comment. I looked at them. Most of the papers below are from this document.

Comment (kbuzard) [reply LIT-REVIEW.Rmd:68]: Great! When you've deleted that from the updated version, go ahead and resolve this conversation.

**Comment by Kirs10-Riley** [comment]
@kbuzard Here is the completed Literature Review.

**Comment by kbuzard** [comment]
@Kirs10-Riley It looks great! Thank you. I'm going to merge it into the repo and you can consider this task finished. Great work!

**Pull request merged**